### ISSUE TITLE
[BUG] [Formatter] [nunjucks] djlint 1.28.0 adds extra parenthesis when a macro is called with multiple arguments

### ISSUE THREAD

**Issue by shaileshaanand** [Riverside-Healthcare/djLint#660]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: alpine 3.18 (Docker: python:3.11.3-alpine3.18)
 - Python Version 3.11.3
 - djLint Version 1.28.0
 - template language: nunjucks

## Issue

djlint 1.28.0 adds extra parenthesis when a macro is called with multiple arguments
This issue started from v1.28.0

## How To Reproduce

Setup:
test.njk
```html
<ul>
    <li>{{ foo(1,2) }}</li>
</ul>
```

Expected Behavior: (taken from v1.17.2)

```
djlint --reformat --profile nunjucks ./test.njk
###
Reformatting 1/1 files ━━━━━━━━━━━━00:00    


0 files were updated.
```

Actual Behavior:
```
djlint --reformat --profile nunjucks ./test.njk
###
Reformatting 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00    


test.njk

@@ -1,3 +1,3 @@

 <ul>
-    <li>{{ foo(1,2) }}</li>
+    <li>{{ foo((1, 2)) }}</li>
 </ul>

1 file was updated.
```

**Comment by shaileshaanand** [comment]
I ran a git bisect and https://github.com/Riverside-Healthcare/djLint/commit/d446efc520c2f7ffec4e418b8a8d4b4a88ba45b1 seems to have introduced this issue
